remote-access/ssh/unix.md

```diff
@@ -18,7 +18,7 @@ When the connection works you will see a security/authenticity warning. Type `ye
 
 In the event your Pi has taken the IP address of a device to which your computer has connected before (even if this was on another network), you may be given a warning and asked to clear the record from your list of known devices. Following this instruction and trying the `ssh` command again should be successful.
 
-Next you will be prompted for the password for the `pi` login: on Raspbian the default password is `raspberry`. You should now be able to see the Raspberry Pi prompt, which will be identical to the one found on the Raspberry Pi itself.
+Next you will be prompted for the password for the `pi` login: on Raspbian the default password is `raspberry`. It is highly recomended, for security reasons, to change the default password on the Raspberry Pi. You should now be able to see the Raspberry Pi prompt, which will be identical to the one found on the Raspberry Pi itself.
```

Comment:
recomended -> recommended

Perhaps this should include a link to a page (e.g. `configuration/security.md`) where it's explained _how_ to change the default password?